about.html

```diff
@@ -0,0 +1,150 @@
+        </ul>
+    </nav>
+
+    <section id="section1" class="section">
+        <nav class="top-bar">      
+            <ul>
+                <li>
+                    <a href="#">
+                        <i class="fab fa-facebook-f"></i>
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        <i class="fab fa-twitter"></i>
+                    </a>
+                </li>
+                <li>
+                    <a class="link-text" href="#">English</a>
+                </li>
+                <li>
+                    <a class="link-text" href="#">My page</a>
+                </li>
+                <li class="logout">
+                    <a class="link-text" href="#">Logout</a>
+                </li>
+            </ul>
+        </nav>
+        <nav class="nav-bar">
+            <a href="home.html">
+                <img class="logo" src="images/logo/logo_no_bg.png" alt="logo">
+            </a>            
+            <ul class="nav-links">
+                <li>
+                    <a href="#">About</a>
+                </li>
+                <li>
+                    <a href="#">Program</a>
+                </li>
+                <li>
+                    <a href="#">Join</a>
+                </li>
+                <li>
+                    <a href="#">Sponsor</a>
+                </li>
+                <li>
+                    <a href="#">News</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="menu-icon">
+            <i class="fas fa-bars"></i>
+        </div>
+        <div class="content">
+            <h2 class="header line1 color1">"Hello! Newbies"</h2>
+            <h1 class="header line2 color1 uppercase">
+                Find the best courses about Web Development, here in one place
+            </h1>
+            <div class="divider1"></div>
+            <p class="color2">
+                The most accesible way to become a software developer and land a jod that will change you life, here in this 
+                Web Site you are going to find the best resources, you need to learn to become a competitive software developer,
+                interactive content, videos, quizes and the best teachers that will help in this new journey.
+            </p>
+            <p class="email-p color2">
+                Please contact us per Email for any futher question about WebDev Couses
+            </p>
+            <a class="email color2" href="#">webdevcourses@gmail.com</a>
+        </div>    
+    </section>
+
+    <section id="section2" class="section">
+        <h2>Main Program</h2>
+        <div class="divider1"></div>
+        <p>Pay once and get acces for live, even if you already have completed you program you can turn back to us and we will be happy to help you.</p>
+        <div class="cont-logo">            
+            <img class="logo" src="images/logo/logo_no_bg.png" alt="logo">
+        </div>
+    </section>
```

Comment:
- [ ] After the logo, it should contain two pictures. Please add it. Pictures can be different, but layout, color, spacing has to be the same.

![image](https://user-images.githubusercontent.com/38635158/148625325-d5ab8814-4a60-4299-970e-878146842175.png)